cluster/manifests/kube-node-ready/service.yaml

```diff
@@ -0,0 +1,19 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: kube-node-ready
+  namespace: kube-system
+  labels:
+    application: kube-node-ready
+    kubernetes.io/cluster-service: "true"
+    kubernetes.io/name: "kube-node-ready"
+  annotations:
+    service.beta.kubernetes.io/external-traffic: OnlyLocal
+spec:
+  type: NodePort
+  ports:
+    - port: 80
+      nodePort: 30797
```

Comment:
but since the target port is `8080` it should maybe be `30080`. It should be a number that makes sense because it will stick.